istio.io/configmap-nginx.yaml

```diff
@@ -112,6 +113,11 @@ data:
         # the github side are to the vanity domain.  DNS points the vanity
         # domain at this nginx.
         location / {
+          # Redirect all version prefixed URLs for non-documentation to
+          # the latest and greatest version of the site, e.g. faq,
+          # blogs.
+          rewrite ^/v-[0-9]+\.[0-9]+(/?.*)$ https://istio.io$1 last;
```

Comment:
it'd be safer anyway to move the / out of the paren and put it also in https://istio.io/$1